frontend/src/components/regions/DueTimeFormModal/Presentation.js

```diff
@@ -1,41 +1,40 @@
-import React from 'react';
-import { makeStyles } from '@material-ui/core/styles';
-import {
-  TextField,
-  Button,
-  Grid,
-  Paper,
-} from "@material-ui/core";
-import Modal from '../../widgets/Modal';
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import { TextField, Button, Grid, Paper } from "@material-ui/core";
+import Modal from "../../widgets/Modal";
+import CloseIcon from "@material-ui/icons/Close";
 
 const useStyles = makeStyles({
-
-    dueTimeButton: {
-        marginTop: "0.3rem",
-        marginRight: "0.3rem",
-    },
-
-    paper: {
-      padding: "1rem"
-    }
+  dueTimeButton: {
+    marginTop: "0.3rem",
+    marginRight: "0.3rem",
+  },
+  paper: {
+    padding: "1rem",
+  },
+  exitIcon: {
+    position: "relative", 
+    top: 0, 
+    left: "200px"
+  }
 });
 
-export default ({
-  card,
-  cardId,
-  handleClose
-}) => {
+export default ({ card, cardId, handleClose }) => {
   const classes = useStyles();
   const dueTime = card.dueTime;
+
   /* dueTime is either null or something like this: "2021-06-14T09:58:28Z".
      If dueTime has a value, then we strip out the seconds, 
      hence dueTime.split('').slice(0, 16).join('') is used */
-  const defaultValue = dueTime ? dueTime.split('').slice(0, 16).join('') : "";
+  const defaultValue = dueTime ? dueTime.split("").slice(0, 16).join("") : "";
   if (cardId)
     return (
       <Modal open={true} onClose={handleClose}>
         <Paper className={classes.paper}>
-          {card ?
+          <div className={classes.exitIcon}>
```

Comment:
This is nice, but I suspect all modals will need consistent exit icons. 

Would it make sense to add it here rather:  "../../widgets/Modal";

Also, can this be a button rather than a div? 

If it walks like a button and quacks like a button we should use a button component

Reply:
I know we are using old MUI, but iconbutton might work: https://mui.com/components/buttons/#upload-button